code/modules/mining/mine_items.dm

```diff
@@ -1353,3 +1353,88 @@ var/list/total_extraction_beacons = list()
 	for(var/turf/simulated/mineral/M in range(7,drill_loc))
 		if(prob(75))
 			M.GetDrilled(1)
+
+/****************Himeo Voidsuit Kit*****************/
+/obj/item/himeo_kit
+	name = "himeo voidsuit kit"
+	contained_sprite = TRUE
+	icon = 'icons/obj/mining_contained.dmi'
+	icon_state = "himeo_kit"
+	item_state = "himeo_kit"
+	desc = "A simple cardboard box containing the requisition forms, permits, and decal kits for a Himean voidsuit."
+	desc_fluff = "As part of a cost-cutting and productivity-enhancing initiative, NanoTrasen has authorized a number of Himean Type-76 'Fish Fur'\
+	for use by miners originating from the planet. Most of these suits are assembled in Cannington and painstakingly optimized on-site by their\
+	individual operator leading to a large trail of red tape as NanoTrasen is forced to inspect these suits to ensure their safety."
+	desc_info = "In order to convert a mining voidsuit into a Himean voidsuit, simply click on this box with a voidsuit or helmet in hand.\
+	The same process can be used to convert a Himean voidsuit back into a regular voidsuit. Make sure not to have a helmet or tank in the suit\
+	or else it will be deleted."
+	w_class = ITEMSIZE_SMALL
+
+/obj/item/himeo_kit/attackby(obj/item/W as obj, mob/user as mob)
+	if(istype(W, /obj/item/clothing/suit/space/void/mining))
+		var/obj/item/clothing/suit/space/void/mining/M = W
+		var/obj/item/clothing/shoes/magboots/B = M.boots
+		if(B)
+			B.canremove = TRUE
+			B.forceMove(get_turf(src))
+			M.boots = null
+		var/obj/item/clothing/head/helmet/H = M.helmet
+		if(H)
+			H.canremove = TRUE
+			H.forceMove(get_turf(src))
+			M.helmet = null
+		var/obj/item/tank/T = M.tank
+		if(T)
+			T.canremove = TRUE
+			T.forceMove(get_turf(src))
+			M.tank = null
+		for(var/obj/accessory in M.accessories)
+			M.remove_accessory(user, accessory)
```

Comment:
This touching `canremove` smells dubious. _Very_ dubious. This paint kit likely should not work on voidsuits which cannot be disassembled (for whatever reason). And as the code is constructed at the moment, that is possible. So basically, leave the deconstruction of the voidsuit up to the person. And just error out when there are pieces still attached to it.